client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +179,12 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<CompactFormToggle
+					checked={ fields[ 'gravatar-hovercards' ] }
+					disabled={ isRequestingSettings || isSavingSettings }
+					onChange={ handleAutosavingToggle( 'gravatar-hovercards' ) }>
+					<span>{ translate( 'Enable pop-up business cards over commenters’ Gravatars' ) }</span>
```

Comment:
I don't think the wrapping `<span>` are necessary anymore. They were needed for historical reasons, but since we've fixed the support for labels for toggles and compact toggles, `span`s can go away.